lib/publiccloud/azure.pm

```diff
@@ -475,7 +499,7 @@ sub terraform_apply {
     my ($self, %args) = @_;
     $args{vars} //= {};
     my $offer = get_var("PUBLIC_CLOUD_AZURE_OFFER");
-    my $sku = get_var("PUBLIC_CLOUD_AZURE_SKU");
+    my $sku = az_sku("");
```

Comment:
An `empty` default is used here as input, to reproduce same behavior as original code L#478 and avoid regression.
See also `subtest az_sku` passed in CI unit test `t/13_...`